backend/apps/matching/tables/approval.py

```diff
@@ -0,0 +1,53 @@
+import django_tables2 as tables
+from django_tables2 import TemplateColumn
+
+from apps.matching.models import Participant
+from apps.matching.src.dual_factory import instanciate_for_participants
+
+approval_button = '<form>{% csrf_token %}<input type="hidden" value="{{record.uuid}}" name="uuid"> <button type="submit" name="change_approval" formmethod="post" class="btn btn-sm {% if record.is_approved %} btn-warning {% else %} btn-success {%endif%}">{% if record.is_approved %}Disapprove{% else %}  Approve {%endif%}</button></form>'
+info_button = (
+    '<a class="btn btn-info btn-sm" href="/matching/hospital_view/{{record.uuid }}">More Info </a>'
```

Comment:
This should redirect to a different link or include a TODO with a linked github issue